src/_includes/config/es-elasticsearch-magento.md

```diff
@@ -2,10 +2,8 @@ This section discusses the minimum settings you must choose to test Elasticsearc
 For additional details about configuring Elasticsearch, see the [{{site.data.var.ee}} User Guide](http://docs.magento.com/m2/ee/user_guide/catalog/search-elasticsearch.html).
 
 {:.bs-callout-warning}
-Magento 2.3.1 adds support for Elasticsearch 6.x, and it is enabled by default.
-Magento still provides modules for Elasticsearch 2.x and 5.x, but these must be enabled in order to use these versions.
-Elasticsearch 2.x is still available but strongly discouraged. Versions 2.x and 5.x are [End of Life](https://www.elastic.co/support/eol).
-If you must run Elasticsearch 2.x or 5.x with Magento 2.3.1, you must downgrade the Elasticsearch client to an older version.
+Magento 2.3.5 adds support for Elasticsearch 7.x.x and 6.8.x.
```

Comment:
Is there somewhere that lists all the versions Magento supports, vs what versions this particular release adds?